src/style.scss

```
@@ -0,0 +1,127 @@
+$main-color: #060b35 ;
+$secondary-color: #616070 ;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Manrope", sans-serif;
```

```suggestion
  font-family: "Roboto", sans-serif;
```
looks like redundant Manrope font family